azure/worker/tests/functions/return_http_404/function.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Nit: Move HttpTrigger tests to folder: HttpTrigger.  Will make it easier to navigate when you add more tests for other triggers

Reply:
Good idea! I'll move them in a separate PR.